centreon/www/class/centreonLDAP.class.php

```diff
@@ -539,7 +538,6 @@ public function listUserForGroup($groupdn): array
 
             return [];
         }
-        $groupdn = str_replace('\\', '\\\\', $groupdn);
```

Comment:
Are you sure removing this has not incidence ?

Reply:
But I see some tests are broken ... will double check